samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -442,6 +442,8 @@
   // Rubrics
   @Getter @Setter
   private String rbcsToken;
+  @Getter @Setter
+  private String rbcAssociation;
```

Comment:
I'd just call this rubricAssociation. It's not much longer and makes more sense to non rubrics devs.